plugins/georeplication/rest.go

```diff
@@ -601,8 +601,7 @@ func checkConfig(name string, value string) error {
 	if value != "" {
 		args = append(args, "--value", value)
 	}
-	_, err := exec.Command(gsyncdCommand, args...).Output()
-	return err
+	return utils.ExecuteCommandRun(gsyncdCommand, args...)
```

Comment:
utils.ExecuteCommandRun(gsyncdCommand, args...) returns two arguments while checkConfig is supposed to return just one argument.

Reply:
`utils.ExecuteCommandRun` returns only one value. `utils.ExecuteCommandOutput` returns two